### ISSUE TITLE
Holding down diagonal only repeats one key event

### ISSUE THREAD

**Issue by norman-r** [ian57/Raspicade-Retrogame-2Player-BPlus#1]
Hi,

Cheers for your work so far. I just noticed one small issue with retrogame: As title says, when you hold down a diagonal key (say you're in SF2 and want to protect against a low kick) , the down and left key events are sent only once, then the last key only is repeated. No big deal for most games but it makes it pretty hard to play games such as SF2. Any fix would be highly appreciated :)

Thx

Norman

**Comment by ian57** [comment]
Yes I think that's te way  retrogame is working : only the last character is repeated. I need to modify the the source code to store the list of down keys... But no time for this actually. Soory
